api/camunda/camunda-openapi.yaml

```diff
@@ -1889,9 +1889,9 @@ components:
         endDate:
           type: string
           format: date-time
-        bpmnProcessId:
+        processDefinitionId:
```

Comment:
@christinaausley I'm not sure we can make this change here - this is the underlying specification and describes the actual operation of the API.